app/controllers/pages_controller.rb

```diff
@@ -1,4 +1,5 @@
 class PagesController < ApplicationController
+  skip_before_action :authenticate_user!, only: :home
```

Comment:
I think we need to skip also on boats collection

Reply:
agree. Everyone should be able to view boats index, and show. So far we do not yet have boats controller in place though, it will come as the next step

Reply:
how can we mark it as to do? Trello?